src/app/views/UserSettings.svelte

```diff
@@ -48,6 +48,13 @@
           Allows {appName} to authenticate with relays that have access controls automatically.
         </p>
       </FieldInline>
+      <Field label="Media provider URL">
+        <Input bind:value={settings.mediaprovider_url}>
+          <i slot="before" class="fa-solid fa-server" />
+        </Input>
+        <p slot="info">
+          Enter a custom url for {appName}'s media provider server (BETA)</p>
```

Comment:
No need to mark as beta, as long as it works

Reply:
Let's call this "Upload Provider URL" rather than media provider, since the user will be downloading images from other servers. The setting/env var can just be called `nip96_url`

Reply:
done